src/style.css

```diff
@@ -1 +1,54 @@
 /* Styles go here */
+
+@font-face {
+  font-family: "Roboto" sans-serif;
+  src: url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+}
+
+a {
+  font-weight: 500;
+  font-family: sans-serif;
+  line-height: 30px;
+}
+
+.nav-bar {
+  display: flex;
+  flex-direction: row;
+  width: auto;
+  justify-content: flex-end;
+  box-shadow: 0.1px 2px 4px rgba(0, 0, 0.05);
+  color: black;
+  list-style-type: none;
+  padding-right: 50px;
+}
+
+.nav-bar:hover {
+  border: solid 3px #00acdc;
+}
```

Comment:
<img width="819" alt="image" src="https://user-images.githubusercontent.com/102293862/216016528-0192f035-55af-4699-8a44-d7a29ffe1c94.png">
that's why. You shouldn't add border to .nav-bar on hover